### ISSUE TITLE
JOSS paper: please describe the deep learning models

### ISSUE THREAD

**Issue by dbuscombe-usgs** [yotarazona/scikit-eo#15]
ping https://github.com/openjournals/joss-reviews/issues/6692

I think it is great that you have provided a deep learning model for landcover classification, thanks for that. Notwithstanding the fact that I haven't yet been able to run the provided notebook (due to https://github.com/yotarazona/scikit-eo/issues/12), I was wondering if you could provide more details about the model, the data it is trained with (it is trained?), the classes it is designed to predict, and other important details, like what data bands it uses for prediction, and how it determines class (softmax?).

There is no mention of those details on the website https://yotarazona.github.io/scikit-eo/deeplearning/ nor the paper. It would appear this is a model that is trained on the fly, rather than pretrained, with weights downloaded from somewhere? Am I right? Also, what makes it 'deep' learning as opposed to machine learning. Those details would be great in the paper and on the website

**Comment by yotarazona** [comment]
Hi @dbuscombe-usgs, sorry for the delay!. This issue was cover in the new version of the paper: https://github.com/yotarazona/scikit-eo/actions/runs/9550361620. Details of the Deep Learning model was added in the notebook tutorial number [11](https://github.com/yotarazona/scikit-eo/blob/main/examples/11_Deep_Learning_Classification_FullyConnected.ipynb).

Thanks a lot for your suggestions which was used to improve ```scikit-eo```.

**Comment by dbuscombe-usgs** [comment]
Thanks
